src/main/app/fee/fee-lookup-api.ts

```diff
@@ -0,0 +1,39 @@
+import axios, { AxiosResponse } from 'axios';
+import config from 'config';
+import { LoggerInstance } from 'winston';
+
+export type Fee = {
+  code: string;
+  description: string;
+  version: string;
+  fee_amount: string;
+};
+
+export const getFee = async (logger: LoggerInstance): Promise<Fee> => {
+  try {
+    const response: AxiosResponse<Fee> = await axios.get(config.get('services.feeLookup.url'), {
+      headers: {
+        accept: 'application/json',
+      },
+      params: {
+        application_type: 'all',
+        channel: 'default',
+        event: 'issue',
+        jurisdiction1: 'family',
+        jurisdiction2: 'family court',
+        keyword: 'ApplyAdoption',
```

Comment:
If keyword is going to decide the type of fee, can we add it as a function argument
for now you can default it to `ApplyAdoption`

Reply:
Need to check for other fee and params and do necessary changes when will work on other fee task